kernel-modules/BLOCKLIST

```diff
@@ -65,7 +65,6 @@
 6.2.* 2.1.0 *
 6.2.* 2.0.1 *
 # TODO(ROX-16705) - 6.3/6.4 kernel compilation errors
-6.3.*
 6.4.*
```

Comment:
You can also remove `6.4.*` from this list, I tested a 6.4 kernel on Fedora locally and the CO-RE BPF probe worked like a charm